views/includes/fixed-footer.pug

```diff
@@ -1,29 +1,15 @@
 div(class='eop')
-  p.discover-heading.sans-serif Discover The Realm of Computer Science Topics!
+  p.discover-heading.sans-serif Codex is based off of the website Medium! Check out our Github links down below
   div(class='tag-grid')
-  button(class='tag-button') Programming
-  button(class='tag-button') Python
-  button(class='tag-button') java
-  button(class='tag-button') C
-  button(class='tag-button') Game programming
-  button(class='tag-button') Data Science
-  button(class='tag-button') Machine Learning
-  button(class='tag-button') Bootcamp
-  button(class='tag-button') arm
+    img(src="./images/codexLogoShort.png", alt="")
   ul.footer-links.sans-serif
     li
-      a(href="") Help
+      a(href="") Justin
     li
-      a(href="") Status
+      a(href="") Kyle
     li
-      a(href="") Writers
+      a(href="") Ricky
     li
-      a(href="") Blog
-    li
-      a(href="") Careers
-    li
-      a(href="") Privacy
-    li
-      a(href="") Terms
+      a(href="") Ken
```

Comment:
Add the following URLs for the hrefs:

Justin Russo - https://github.com/justinrusso
Ken Julian - https://github.com/kenjulian
Kyle Huang - https://github.com/kvh8899
Ricky Tang - https://github.com/rickythewriter